vercel.json

```diff
@@ -3161,6 +3161,36 @@
       "source": "/docs/integrations/redshift/redshift-push-to-clickhouse",
       "destination": "/docs/integrations/redshift",
       "permanent": true
+    },
+    {
+      "source": "/docs/guides/developer/lightweght-delete",
+      "destination": "/docs/guides/developer/lightweight-delete",
+      "permanent": true
+    },
+    {
+      "source": "/docs/integrations/java/jdbc-v2",
+      "destination": "/docs/integrations/language-clients/java/jdbc",
+      "permanent": true
+    },
+    {
+      "source": "/docs/integrations/java/client-v2",
+      "destination": "/docs/integrations/language-clients/java/jdbc",
+      "permanent": true
+    },
+    {
+      "source": "/docs/interfaces/formats/Protobuf/Protobuf",
+      "destination": "/docs/interfaces/formats/Protobuf",
+      "permanent": true
+    },
+    {
+      "source": "/docs/get-started/query-endpoints",
+      "destination": "/docs/cloud/get-started/query-endpoints",
+      "permanent": true
+    },
+    {
+      "source": "/docs/getting-started/example-datasets/star-schema/u003C/p",
```

Comment:
this seems unneccessary, its not a valid URL

Reply:
true but it's being linked from somewhere - 32 hits in the last 24 hours. Can't hurt to direct the traffic to the correct page.

Reply:
maybe but id rather the 404s be detected at source vs encouraging this by redirecting

Reply:
Fixed